examples/python/gesture_detection/README.md

```diff
@@ -124,7 +124,6 @@ rr.log(
 ## Run the code
 To run this example, make sure you have the Rerun repository checked out and the latest SDK installed:
 ```bash
-# Setup
```

Comment:
removed all of these `# Setup` comments since they don't add any value (and were in the way)